app/poros/forecast.rb

```diff
@@ -0,0 +1,32 @@
+class Forecast
+  def initialize(forecast_data)
+    @forecast_data = forecast_data
+    @current_weather = current_weather(forecast_data)
+    @daily_forecast = daily_forecast(forecast_data)
+    @hourly_forecast = hourly_forecast(forecast_data)
+  end
+
+  def current_weather(forecast_data)
+    CurrentWeather.new(forecast_data).current_conditions
+  end
+
+  def daily_forecast(forecast_data)
+    DailyForecast.new(daily_data).next_five_days
+  end
+
+  def hourly_forecast(forecast_data)
+    HourlyForecast.new(hourly_data).next_eight_hours
```

Comment:
Same as above. If you are using the `hourly_data` method, you don't need to pass forecast_data through as an argument. And if you map through the hourly data at this level and pass a single hours data into the HourlyForecast PORO you can get rid of `.next_eight_hours`. This is similar to my comment about DailyForecast above.